application/services/telegram.py

```diff
@@ -0,0 +1,90 @@
+import requests
+import logging
+
+from telegram import Update
+from telegram.ext import ApplicationBuilder, ContextTypes, CommandHandler, filters
+
+from application.services import common
+from application.services import security as vault
+from application.services import sms_service
+
+logging.basicConfig(
+    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+    level=logging.INFO
+)
+
+ALLOWED_CHAT_IDS = common.get_active_users_value("chat_id")
```

Comment:
This may eventually get stale, would be better to update each call :D

Reply:
Facts! 😃